Xamarin.Forms.Platform.Android/FastRenderers/AccessibilityProvider.cs

```diff
@@ -137,19 +136,15 @@ bool SetHint(string hint = GetFromElement)
 				return true;
 			}
 
-			if (_defaultHint == null)
-			{
-				_defaultHint = textView.Hint;
-			}
+			string accessibilityName = (string)Element.GetValue(Accessibility.NameProperty);
+			string accessibilityHint = (string)Element.GetValue(Accessibility.HintProperty);
 
-			string value = hint;
-			if (value == GetFromElement)
+			if (string.IsNullOrWhiteSpace(accessibilityName) && string.IsNullOrWhiteSpace(accessibilityHint))
 			{
-				value = string.Join(". ", (string)Element.GetValue(Accessibility.NameProperty),
-					(string)Element.GetValue(Accessibility.HintProperty));
+				return true;
 			}
 
-			textView.Hint = !string.IsNullOrWhiteSpace(value) ? value : _defaultHint;
+			textView.Hint = string.Join(". ", accessibilityName, accessibilityHint);
```

Comment:
Idont know what to expect if only one value is provided. Maybe someone at microsoft/xamarin (you?) has that answer, from the original code its not possible to understand how its supposed to work :P.  If the join is a no go we should probably clean it up in the standard renderer aswell.  Il update my code again. The meaning of this PR is solely to not give TextViews a hint of ". "